src/System.Diagnostics.FileVersionInfo/tests/System.Diagnostics.FileVersionInfo.Tests/FileVersionInfoTest.cs

```diff
@@ -102,6 +102,14 @@ public void FileVersionInfo_NonExistentFile_FileNotFound()
                 FileVersionInfo.GetVersionInfo(Path.Combine(Directory.GetCurrentDirectory(), TestNotFoundFileName)));
         }
 
+        [Fact]
+        public void FileVersionInfo_RelativePath_ArgumentException()
+        {
+            File.WriteAllText("kernelbase.dll", "bogus kernelbase.dll");
```

Comment:
Tests mustn't write to current directory. Follow pattern elsewhere to use temp path

Reply:
Do you have an example? Also, will it be possible to get a relative path to the temp file? Having the absolute path would defeat the purpose